.github/actions/check-dependent-jobs/action.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+name: check-dependent-jobs
+description: |
+  Evaluates the combined the status results of the provided needs context.
+inputs:
+  needs:
+    required: true
+    description: needs context as JSON string
+outputs:
+  isSuccess:
+    description: the evaluated result of all provided jobs in the needs context.
+    value: ${{ steps.test.outputs.isSuccess }}
+runs:
+  using: composite
+  steps:
+    - id: test
+      run: |
+        RESULT=$(echo '${{ inputs.needs }}' | jq -s 'map(.[].result) | all(.=="success")')
+        echo "isSuccess=${RESULT}" >> $GITHUB_OUTPUT
```

Comment:
As far as I see `true`/`false` works quite well with the *nix commands as they are recognised commands, while `0`/`1` are not the case.

Do you think converting those values to `true`/`false` could work nicely with the consumers? IIUC, in the example section defined in the README.md uses ` - run: ${{ steps.is_success.outputs.isSuccess }}` so, it will fail in both cases if `0`/`1` while it will work fine with `true`/`false`.

Reply:
the value of `$RESULT` should be `true` or `false` already. Or am I misunderstanding?

Reply:
![image](https://user-images.githubusercontent.com/16325797/223669796-b2a851da-4e55-43e5-85d6-74ec2a18e952.png)

Source: https://github.com/elastic/apm-pipeline-library/actions/runs/4362622323/jobs/7627736701

Reply:
ahh!! I misread the comment `# should exit with 1 or 0.`, my bad! Then all is good!

Reply:
FYI, also renamed the ids: https://github.com/elastic/apm-pipeline-library/pull/2126/files/1659d78dc23d38bf3af0b7c4b20c0c056979e656..8ac2a651d5d842476b515d8b7ead4c264b411485